### PR TITLE
Fix maze_runner.wbt and obstacle_avoidance.wbt

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
1. Fix appearance of window's wall that were mistakenly set to white instead of green (as the other walls).
  Note that even if the `PBRAppearance` node for the walls and the window's walls is exactly the same, in Webots the resulting color is slightly different. This difference is not visible in THREEjs

2. Disable the `castShadows` flag for walls, windows and some other objects attached to the wall that are not supposed to cast shadows inside the room.
  Given that we the room is illuminated using a DirectionalLight, if not specified the walls will cast shadows inside the room in THREEjs. In Webots walls strangely don't cast shadow even  if the flag is not unset, but the Blackboard object does.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:-1: 

I forgot to disable casting shadows for the lights.

File: projects/objects/solids/protos/SolidBox.proto [hunk 1]
```diff
@@ -13,7 +13,8 @@ PROTO SolidBox [
   field SFVec3f    size            2 2 2                                                                                                         # Defines the size of the box.                                                                                                             # Defines the number of polygons used to represent the box and so its resolution.
   field SFString   contactMaterial "default"                                                                                                     # Is `Solid.contactMaterial`.
   field SFNode     appearance      PBRAppearance { baseColorMap ImageTexture { url [ "textures/tagged_wall.jpg" ] } metalness 0 roughness 0.5 }  # Defines the appearance of the box.
-  field SFNode     physics         NULL                                                                                                          # Is `Solid.physics`.
+  field SFNode     physics         NULL      
+  field SFBool     castShadows     TRUE              # Defines if this object cast shadows.                                                                                                    # Is `Solid.physics`.
```
Comment (DavidMansolino) [inline projects/objects/solids/protos/SolidBox.proto]: The two field descriptions are on the same line.

**Comment by stefaniapedrazzi** [comment]
test me

**Comment by stefaniapedrazzi** [comment]
test me

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by DavidMansolino** [comment]
:eyes:

**Pull request merged**